docs/src/architecture/accounts.md

```diff
@@ -44,14 +44,100 @@ Merkle tree as follows:
 An account vault can be reduced to a single hash which is the root of the sparse Merkle tree.
 
 ### Code
-Interface for accounts. In Miden every account is a smart contract. It has an interface that exposes functions that can be called by note scripts. Functions exposed by the account have the following properties:
+Interface for accounts. In Miden every account is a smart contract. It has an interface that exposes functions that can be called by note and transaction scripts. Users cannot call those functions directly. 
+
+Functions exposed by the account have the following properties:
 
 * Functions are actually roots of [Miden program MASTs](https://0xpolygonmiden.github.io/miden-vm/user_docs/assembly/main.html) (i.e., a 32-byte hash). Thus, function identifier is a commitment to the code which is executed when a function is invoked.
 * Only account functions have mutable access to an account's storage and vault. Therefore, the only way to modify an account's internal state is through one of the account's functions.
 * Account functions can take parameters and can create new notes.
 
 *Note: Since code in Miden is expresed as MAST, every function is a commitment to the underlying code. The code cannot change unnoticed to the user because its hash would change. Behind any MAST root there can only be `256` functions*
 
+#### Example Account Code
+Currently, Miden provides two standard implementations for account code. 
+
+##### Basic user account (Regular updatable account)
+There is a standard for a basic user account. It exposes three functions via its interface.
+<details>
+  <summary>What to see the code?</summary>
+
+  ```
+    use.miden::contracts::wallets::basic->basic_wallet
+    use.miden::contracts::auth::basic
+
+    export.basic_wallet::receive_asset
+    export.basic_wallet::send_asset
+    export.basic::auth_tx_rpo_falcon512
+  ```
+</details>
+
+[Note scripts](https://0xpolygonmiden.github.io/miden-base/architecture/notes.html#note-scripts) or [transaction scripts](https://0xpolygonmiden.github.io/miden-base/transactions/transaction-kernel.html#the-transaction-script-processing) can call `receive_asset` and `send_asset` and in doing so, the account can receive and send assets. Transaction scripts can also call `auth_tx_rpo_falcon512` and authenticate the transaction. It is important to know, that without correct authentication, i.e. knowing the correct private Falcon key, a note cannot successfully invoke receive and send asset.
```

Comment:
As added in the prev comment, maybe make it more future proof by removing the reference to Falcon